Source/Blazorise/Components/FocusTrap/FocusTrap.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@namespace Blazorise
 @inherits BaseComponent
-<div @ref="@containerRef" @onkeydown="@OnKeyPressesHandler" @onkeyup="@OnKeyPressesHandler">
+<div @ref="@containerRef" @onkeydown="@OnKeyPressedHandler" @onkeyup="@OnKeyPressedHandler">
```

Comment:
Why both keydown and keyup, will this not double trigger the Handler?

Reply:
Yes is it true that it will raise two events for the same key; But, I assume it is OK as Chris hopefully knew better at the time 😅